azext_iot/_help.py

```diff
@@ -61,23 +67,25 @@
 helps['iot hub device-identity update'] = """
     type: command
     short-summary: Update an IoT Hub device.
-    long-summary: Use --set followed by property assignments for updating your device.
+    long-summary: Use --set followed by property assignments for updating a device.
                   Leverage parameters returned from 'iot hub device-identity show'.
     examples:
     - name: Turn on edge capabilities for device
       text: >
-        az iot hub device-identity update -d mydevice -n myhub --set capabilities.iotEdge=true
+        az iot hub device-identity update -d [Device ID] -n [IoTHub Name] 
+        --set capabilities.iotEdge=true
     - name: Disable device status
       text: >
-        az iot hub device-identity update -d mydevice -n myhub --set status=disabled
+        az iot hub device-identity update -d [Devie ID] -n [IoTHub Name] --set status=disabled
```

Comment:
Small typo here.

Reply:
fixed in commit: https://github.com/Azure/azure-iot-cli-extension/pull/4/commits/9647fc683a6f35fb880a1bcfdea8e2fefa19391b